test/toucan/db_test.clj

```diff
@@ -169,9 +170,29 @@
 
 ;; TODO - Test simple-delete!
 
-;; TODO - Test simple-insert-many!
+;; Test simple-insert-many!
+(expect
+ '(4 5)
+ (test/with-clean-db
+   (db/simple-insert-many! User [{:first-name "Grass" :last-name #sql/call [:upper "Hopper"]}
+                                 {:first-name "Ko" :last-name "Libri"}])))
 
-;; TODO - Test insert-many!
+(expect
+ '(5)
+ (test/with-clean-db
+   (db/simple-insert-many! Category [{:name "seafood" :parent-category-id 100}])))
+
+;; Test insert-many!
+(expect
+ '(4 5)
+ (test/with-clean-db
+   (db/insert-many! User [{:first-name "Grass" :last-name #sql/call [:upper "Hopper"]}
+                          {:first-name "Ko" :last-name "Libri"}])))
+
+(expect
```

Comment:
e.g. why do we expect this test to fail. It's not clear to me by looking at it why it's supposed to fail